azkaban-common/src/test/resources/image_management/image_type_rampups.json

```diff
@@ -1,42 +1,42 @@
 {
   "spark_job": [
-    {
-      "imageVersion": "1.1.2",
-      "rampupPercentage": "70"
-    },
     {
       "imageVersion": "1.1.1",
       "rampupPercentage": "30"
+    },
+    {
+      "imageVersion": "1.1.2",
```

Comment:
Can you please add three or more versions in one of the test cases? Add different order like
1.1.3
1.1.1
1.1.2
This will cover that ordering is not required.

Reply:
This JSON is what is created after the query, that is why it needs to be sorted to mimic the query result. There is no change other than sort order of image version.
I can add 1 more version in a few cases.

Reply:
That sounds good. Can you please create ramp up plan with three and order that I mentioned and then verify this order?

Reply:
Done.